ddtrace/ext/ci.py

```diff
@@ -542,4 +559,5 @@ def extract_buddy(env):
     ("TRAVIS", extract_travis),
     ("BITRISE_BUILD_SLUG", extract_bitrise),
     ("BUDDY", extract_buddy),
+    ("CODEBUILD_INITIATOR", extract_awscodepipeline),
```

Comment:
I think the spec says that we should not only check for CODEBUILD_INITIATOR env variable but also that it starts with `codepipeline `, so I have to refactor the similar logic in the ruby libarry

Reply:
Oh man, 100% missed that. Thank you! Will update.

Reply:
Okay, it's updated. I was originally thinking a refactor would be nice (kind of like you did in https://github.com/DataDog/datadog-ci-rb/pull/52 , @anmarchenko ) but in the end, the behavior is the same: the tags aren't added, so I figured leaving things as is would be simpler.

Reply:
My `git push` failed and I didn't pay attention. 👎 , sorry. It's coming.